defold-spine/src/gui_node_spine.cpp

```diff
@@ -369,6 +369,87 @@ void CancelAnimation(dmGui::HScene scene, dmGui::HNode hnode)
     node->m_Playing = 0;
 }
 
+bool AddSkin(dmGui::HScene scene, dmGui::HNode hnode, dmhash_t skin_id_a, dmhash_t skin_id_b){
+    InternalGuiNode* node = (InternalGuiNode*)dmGui::GetNodeCustomData(scene, hnode);
+    spSkin* skin_a = node->m_SpineScene->m_Skeleton->defaultSkin;
+    spSkin* skin_b = node->m_SpineScene->m_Skeleton->defaultSkin;
+
+    if (skin_id_a)
+    {
+        uint32_t* index = node->m_SpineScene->m_SkinNameToIndex.Get(skin_id_a);
+        if (!index)
+        {
+            dmLogError("No skin '%s' found", dmHashReverseSafe64(skin_id_a));
+            return false;
+        } else {
+            skin_a = node->m_SpineScene->m_Skeleton->skins[*index];
+        }
+    }
+    if (skin_id_b)
+    {
+        uint32_t* index = node->m_SpineScene->m_SkinNameToIndex.Get(skin_id_b);
+        if (!index)
+        {
+            dmLogError("No skin '%s' found", dmHashReverseSafe64(skin_id_b));
+            return false;
+        } else {
+            skin_b = node->m_SpineScene->m_Skeleton->skins[*index];
+        }
```

Comment:
To maintain the same code style it should be 

```
}
else
{
```

But in this case you might as well skip the else since you do an early exit if the index is invalid. Like you do in comp_spine_model.cpp